pelican/plugins/seo/tests/conftest.py

```diff
@@ -195,6 +195,51 @@ def fake_article_multiple_elements():
     )
 
 
+@pytest.fixture()
+def fake_article_multiple_elements_with_seo():
```

Comment:
Why do you need another fixture? Normally with `fake_article`, `fake_article_multiple_elements` and `fake_article_missing_elements` you should have all you need for your test, shouldn't you?

Reply:
Initially I thought that if I change the title and description of one of the existing fixtures it'll break other tests so I added a new one. It seems that though that those properties are never uses anywhere else for `fake_article_multiple_elements` so I change this one and removed the newly added fixture. Let me know please if this is acceptable.